src/DotLiquid/StandardFilters.cs

```diff
@@ -390,12 +363,7 @@ internal static IEnumerable SortInternal(StringComparer stringComparer, object i
             if (!ary.Any())
                 return ary;
 
-#if NETSTANDARD1_3
-            var comparer = new StringAwareObjectComparer(stringComparer);
-#else
             var comparer = stringComparer;

```

Comment:
You could rename the argument `stringComparer` to `comparer` and delete this line.